server/tests/riders.test.ts

```diff
@@ -0,0 +1,261 @@
+import request from 'supertest';
```

Comment:
would recommend renaming file to rider.test.ts to match the source file rider.ts